### ISSUE TITLE
405 Method Not Allowed: MKCOL can only be executed on an unmapped URL.

### ISSUE THREAD

**Issue by chatainsim** [ncw/rclone#3035]
#### What is the problem you are having with rclone?
rclone sync from Google Drive to Webdav (Seafile server)

For some files I have this error:
```2019/03/08 10:12:44 ERROR : APRR/250092755428/Aprr_2018-08-01_FV86152249.pdf: Failed to copy: Update mkParentDir failed: <!DOCTYPE HTML PUBLIC '-//W3C//DTD HTML 4.01//EN' 'http://www.w3.org/TR/html4/strict.dtd'>
<html><head>
  <meta http-equiv='Content-Type' content='text/html; charset=UTF-8'>
  <title>405 Method Not Allowed</title>
</head><body>
  <h1>405 Method Not Allowed</h1>
  <p>405 Method Not Allowed: MKCOL can only be executed on an unmapped URL.</p>
<hr/>
<a href='https://github.com/mar10/wsgidav/'>WsgiDAV/1.2.0.pre</a> - 2019-03-08 10:12:44.012876
</body></html>: 405 Method Not Allowed
```
#### What is your rclone version (output from `rclone version`)
rclone v1.46
- os/arch: linux/amd64
- go version: go1.11.5

#### Which OS you are using and how many bits (eg Windows 7, 64 bit)
Linux 3.16.0-4-amd64 #1 SMP Debian 3.16.51-3 (2017-12-13) x86_64 GNU/Linux
Distributor ID: Debian
Description:    Debian GNU/Linux 8.11 (jessie)
Release:        8
Codename:       jessie

####  Which cloud storage system are you using? (eg Google Drive)
Google Drive to Webdav (Seafile)

#### The command you were trying to run (eg `rclone copy /tmp remote:tmp`)
rclone sync gdrive:securibox Seafile:

#### A log from the command with the `-vv` flag (eg output from `rclone -vv copy /tmp remote:tmp`)
The current command is still running, while it's done, I will add the verbose output here.

**Comment by ncw** [comment]
Can you replicate this with a small test (with some small text files maybe) and post the result when run with `-vv --dump bodies`.

Rclone should be detecting a 405 error from MKCOL already so the above will help to see what exactly is going on.

**Comment by chatainsim** [comment]
Here is the output:
```
2019/03/08 13:18:50 DEBUG : >>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>
2019/03/08 13:18:50 DEBUG : <<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<
2019/03/08 13:18:50 DEBUG : HTTP RESPONSE (req 0xc000ca4e00)
2019/03/08 13:18:50 DEBUG : HTTP/1.1 405 Method Not Allowed
Content-Length: 457
Connection: keep-alive
Content-Type: text/html; charset=utf-8
Date: Fri, 08 Mar 2019 12:18:50 GMT
Server: nginx

<!DOCTYPE HTML PUBLIC '-//W3C//DTD HTML 4.01//EN' 'http://www.w3.org/TR/html4/strict.dtd'>
<html><head>
  <meta http-equiv='Content-Type' content='text/html; charset=UTF-8'>
  <title>405 Method Not Allowed</title>
</head><body>
  <h1>405 Method Not Allowed</h1>
  <p>405 Method Not Allowed: MKCOL can only be executed on an unmapped URL.</p>
<hr/>
<a href='https://github.com/mar10/wsgidav/'>WsgiDAV/1.2.0.pre</a> - 2019-03-08 13:18:50.307935
</body></html>
2019/03/08 13:18:50 DEBUG : <<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<
2019/03/08 13:18:50 DEBUG : Voyages-SNCF/email@gmail.com: Making directory
2019/03/08 13:18:50 DEBUG : >>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>
2019/03/08 13:18:50 DEBUG : HTTP REQUEST (req 0xc000ca5000)
2019/03/08 13:18:50 DEBUG : MKCOL /seafdav/rclone/Voyages-SNCF/email@gmail.com/ HTTP/1.1
Host: box.easter.fr
User-Agent: rclone/v1.46
Authorization: XXXX
Accept-Encoding: gzip
```

**Comment by ncw** [comment]
I think I see what is going on....  Can you try this beta?

https://beta.rclone.org/branch/v1.46.0-060-ge1e46d12-fix-3035-webdav-race-beta/ (uploaded in 15-30 
mins)

**Comment by chatainsim** [comment]
Still some issue:
```
2019/03/08 15:20:38 DEBUG : >>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>
2019/03/08 15:20:38 DEBUG : <<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<
2019/03/08 15:20:38 DEBUG : HTTP RESPONSE (req 0xc000972700)
2019/03/08 15:20:38 DEBUG : HTTP/1.1 405 Method Not Allowed
Content-Length: 457
Connection: keep-alive
Content-Type: text/html; charset=utf-8
Date: Fri, 08 Mar 2019 14:20:38 GMT
Server: nginx

<!DOCTYPE HTML PUBLIC '-//W3C//DTD HTML 4.01//EN' 'http://www.w3.org/TR/html4/strict.dtd'>
<html><head>
  <meta http-equiv='Content-Type' content='text/html; charset=UTF-8'>
  <title>405 Method Not Allowed</title>
</head><body>
  <h1>405 Method Not Allowed</h1>
  <p>405 Method Not Allowed: MKCOL can only be executed on an unmapped URL.</p>
<hr/>
<a href='https://github.com/mar10/wsgidav/'>WsgiDAV/1.2.0.pre</a> - 2019-03-08 15:20:38.957716
</body></html>
2019/03/08 15:20:38 DEBUG : <<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<
2019/03/08 15:20:38 DEBUG : La Poste LRE - Particulier/email@gmail.com: Making directory
2019/03/08 15:20:38 DEBUG : >>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>
2019/03/08 15:20:38 DEBUG : HTTP REQUEST (req 0xc00086f300)
2019/03/08 15:20:38 DEBUG : MKCOL /seafdav/rclone/La%20Poste%20LRE%20-%20Particulier/email@gmail.com/ HTTP/1.1
Host: box.easter.fr
User-Agent: rclone/v1.46.0-060-ge1e46d12-fix-3035-webdav-race-beta
Authorization: XXXX
Accept-Encoding: gzip

2019/03/08 15:20:38 DEBUG : >>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>
2019/03/08 15:20:39 DEBUG : <<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<
2019/03/08 15:20:39 DEBUG : HTTP RESPONSE (req 0xc00086f300)
2019/03/08 15:20:39 DEBUG : HTTP/1.1 405 Method Not Allowed
Content-Length: 457
Connection: keep-alive
Content-Type: text/html; charset=utf-8
Date: Fri, 08 Mar 2019 14:20:39 GMT
Server: nginx

<!DOCTYPE HTML PUBLIC '-//W3C//DTD HTML 4.01//EN' 'http://www.w3.org/TR/html4/strict.dtd'>
<html><head>
  <meta http-equiv='Content-Type' content='text/html; charset=UTF-8'>
  <title>405 Method Not Allowed</title>
</head><body>
  <h1>405 Method Not Allowed</h1>
  <p>405 Method Not Allowed: MKCOL can only be executed on an unmapped URL.</p>
<hr/>
<a href='https://github.com/mar10/wsgidav/'>WsgiDAV/1.2.0.pre</a> - 2019-03-08 15:20:39.347809
</body></html>
2019/03/08 15:20:39 DEBUG : <<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<
2019/03/08 15:20:39 DEBUG : webdav root '': copied 22 directories
2019/03/08 15:20:39 INFO  : Waiting for deletions to finish
2019/03/08 15:20:39 INFO  :
Transferred:             0 / 0 Bytes, -, 0 Bytes/s, ETA -
Errors:                 0
Checks:              1457 / 1457, 100%
Transferred:            0 / 0, -
Elapsed time:     7m50.3s

2019/03/08 15:20:39 DEBUG : 35 go routines active
2019/03/08 15:20:39 DEBUG : rclone: Version "v1.46.0-060-ge1e46d12-fix-3035-webdav-race-beta" finishing with parameters ["rclone" "-vv" "--dump" "bodies" "sync" "gdrive:securibox" "Seafile:"]

```

**Comment by ncw** [comment]
That looked like it worked with 0 errors.  You'll still see the 405 errors in the HTTP dump but rclone is ignoring them correctly now I think

**Comment by chatainsim** [comment]
Ok, let me try with some others data.
I'll keep you in touch.

Thanks

**Comment by chatainsim** [comment]
I confirm, no more errors.
Thank you !

**Comment by ncw** [comment]
Thanks for testing.

I've merged this to master now which means it will be in [the latest beta](https://beta.rclone.org) in 15-30 mins and released in v1.47
